platforms/ios/HelloCordova/Plugins/WizAssetsPlugin/WizAssetsPlugin.m

```diff
@@ -256,10 +256,10 @@ - (void)deleteFiles:(CDVInvokedUrlCommand *)command {
 
 - (void)backgroundDelete:(CDVInvokedUrlCommand *)command {
     @autoreleasepool {
-        NSMutableArray *uris = [[NSMutableArray alloc] initWithArray:command.arguments copyItems:YES];
+        NSArray *uris = command.arguments;
```

Comment:
I guess my assertion was correct that an NSArray would do just fine (compared to a resizable array). I didn't say anything about not copying the items though, but did you see an opportunity here that I missed? My only concern would be some weird multi threaded case (that you would be more suited to identify than me here), where not having a copy would create corrupted reads.